client/dashboard/sites/settings-transfer-site/confirm-new-owner-form.tsx

```diff
@@ -20,19 +19,42 @@ export type ConfirmNewOwnerFormData = {
 	email: string;
 };
 
-const fields: Field< ConfirmNewOwnerFormData >[] = [
+const form = {
+	layout: { type: 'regular' as const },
+	fields: [ 'email' ],
+};
+
+const useFields = ( siteId: number ): Field< ConfirmNewOwnerFormData >[] => [
```

Comment:
This is not a hook. Maybe just `getFields()`?

Reply:
Ok. Done by https://github.com/Automattic/wp-calypso/pull/107230/commits/15497ce0e0baa93046183573736e9bcb958bd686.